web/i18n/fa.yml

```diff
@@ -435,13 +435,13 @@ flexigrid:
     pagestat: نمایش {from} تا {to} از {total} مورد
     outof: از
     findtext: جست‌وجو
-    procmsg: در حال پردازش، لطفاً شکیبا باشید...
+    procmsg: در حال پردازش، لطفاً شکیبا باشید...&#8207;
```

Comment:
Is this correct?

Reply:
That is the right-to-left mark. I putted it to force the "..." to be displayed at the correct place in the string. we could have a smarter way to solve that issue: using css rules
‘html[dir="rtl"] .pPageStat {direction:rtl;}‘
for general and
‘html[dir="rtl"] .pPageStat {direction:ltr;}‘
for mobile view that we have sth like 1..23/140 because of space limit. without the second and by presence of first rule, order of characters in 1..23/140 will be incorrect.
I forgot to add it in the proposed css file and did it this way. yes, this is a bad idea since it only applies to persian.

Reply:
Okay. Please remove it then and I can accept the pull request.